Assets/Scripts/Quests/Quest.cs

```diff
@@ -37,11 +41,26 @@ private enum Location
         private readonly List<Adventurer> _assigned = new List<Adventurer>();
 
         public string Title => title;
+        public string Reward { get; private set; }
         public string Description => description;
         public int Cost { get; private set; }
         public int TurnsLeft { get; private set; }
         public Structure Structure { get; private set; }
         public bool IsActive => TurnsLeft != -1;
+        public bool IsRadiant => QuestLocation is Location.Grid;
```

Comment:
is using 'is' here the same as ==?

Reply:
Yeah, seems safe to do so with enum comparisons.